core/src/actors/blocks_manager/actor.rs

```diff
@@ -137,7 +137,7 @@ impl Actor for BlocksManager {
                         initialize and store a new chain."
                         );
                         // Create a new ChainInfo
```

Comment:
What is a `ChainInfo`? Is it a file? Is it a message? We may want to be more descriptive.

Reply:
You are right, it should be defined [here](https://github.com/witnet/witnet-rust/blob/master/data_structures/src/chain.rs). I will include it in this PR.

Reply:
Great!